rustler_mix/lib/rustler.ex

```diff
@@ -52,12 +47,6 @@ defmodule Rustler do
       - When `Mix.env()` is `:dev` or `:test`, the crate will be compiled in `:debug` mode.
       - When `Mix.env()` is `:prod` or `:bench`, the crate will be compiled in `:release` mode.
 
-    * `:path` - By default, rustler expects the crate to be found in `native/<crate>` in the
-      root of the project. Use this option to override this.
-
```

Comment:
This is not necessary anymore, as we now expect a `Cargo.toml` in the root of the project.